plugins/plugin-docker/che-plugin-docker-machine/src/test/java/org/eclipse/che/plugin/docker/machine/cleaner/DockerAbandonedResourcesCleanerTest.java

```diff
@@ -177,4 +203,40 @@ public void cleanerShouldNotKillAndRemoveContainerIfContainerNameInfoIsEmpty() t
 
         verify(dockerConnector, never()).removeContainer(Matchers.anyObject());
     }
+
+    @Test
+    public void shouldRemoveOnlyAbandonedNetworks() throws IOException {
```

Comment:
Please add other tests, e.g. when containers list fetching fails, when there is a container in matching  network,  when there are several networks including non-matching by name, when network is in additional networks list, when networks list fetching fails, network removal fails, network is in runtimes. 

Please cover your code well. And do not use single test for several test cases